chai/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "chai"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+chrono = "0.4.31"
+chrono-tz = "0.8.4"
+futures-util = "0.3.28"
+trias = { path = "../trias" }
+reqwest = "0.11.22"
+serde = { version = "1.0.188", features = ["derive"] }
+serde_json = "1.0.107"
+specta = { version = "1.0.5", features = ["export"] }
+dotenv = "0.15.0"
+axum = { workspace = true }
+tracing = "0.1.25"
```

Comment:
It's unfortunate that this depends on `reqwest` and `axum`, since ideally it should be independent of what libraries it's used with.